proposals/lip-0055.md

```diff
@@ -66,7 +67,16 @@ As an example, blockchains created with the Lisk SDK that implement the [Random
 
 The schema for the block assets allows each module to include its serialized data individually, which makes the inclusion of module data very flexible. Each module can insert a single entry in the assets. This entry is an object containing a `moduleID` property, indicating the ID of the module handling it, and a generic `data` property that can contain arbitrary serialized data.
 
-Each entry of the block assets is then inserted in a Merkle tree, whose root is included in the block header as the `assetsRoot` property. Inserting the assets root rather than the full assets allows to bound the size of the block header while still authenticating the content of the block assets.
+Each entry of the block assets is then inserted in a Merkle tree, whose root is included in the block header as the `assetRoot` property. Inserting the assets root rather than the full assets allows to bound the size of the block header while still authenticating the content of the block assets.
+
+
+### Failing and Invalid Transactions
```

Comment:
The section is currently missing the rationale of why we add this feature. Maybe we could add 2-3 sentences about computationally expensive transactions that can still be cheaply verified in the transaction pool and during block generation.

Reply:
added a new paragraph